app/views/system/admin/announcements/_announcement.html.slim

```diff
@@ -2,17 +2,21 @@
   h2
     - if announcement.unread?
       input.new_announcement type="hidden"
-      = fa_icon 'envelope'
+        = fa_icon 'envelope'
+
     = announcement.title
+
   - if can? :manage, announcement.class
     div.pull-right
-      = edit_button([:admin, announcement])
-      = delete_button([:admin, announcement])
+      = edit_button(edit_admin_announcement_path(announcement))
+
+      = delete_button(admin_announcement_path(announcement))
```

Comment:
I would rather not have extra space between lines since both of them belong to the same div.